lib/repository.js

```diff
@@ -145,7 +145,13 @@ Repository.prototype.getReference = function(name, callback) {
   }, callback);
 };
 
-Repository.getReferences = function(repo, type, refNamesOnly, callback) {
+Repository.getReferences =
+function(repo, type, refNamesOnly, includeSymbolic, callback) {
+  if (typeof callback === "undefined" &&
```

Comment:
Do we need a typeof check here? That's really only necessary if the variable is possibly undeclared.  Simply do `!callback` or `callback == null`.